src/components/App.js

```diff
@@ -29,7 +30,7 @@ Top level router - routing between an actual form or supporting screens.
 const App = ({ languageSelectorTarget, ...props }) => {
   return (
     <Layout>
-      <LanguageSwitcher target={languageSelectorTarget} />
+      <LanguageSwitcher target={languageSelectorTarget} renderSelector={props.form.translationEnabled} />
```

Comment:
it makes a bit more sense to decide not to render a node in the parent (`App` in this case) if you only pass along that information:

```jsx
{props.form.translationEnabled ? <LanguageSwitcher ... /> : null}
```